### ISSUE TITLE
github deprecation of unauthenticated git clones

### ISSUE THREAD

**Issue by nitrocode** [pre-commit/action#138]
https://github.blog/2021-09-01-improving-git-protocol-security-github/

error
```
 Run pre-commit/action@v2.0.2
 install pre-commit
 /opt/hostedtoolcache/Python/3.10.1/x64/bin/pre-commit run --show-diff-on-failure --color=always --all-files
 [INFO] Initializing environment for git://github.com/antonbabenko/pre-commit-terraform.
 An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'fetch', 'origin', '--tags')
 return code: 128
 expected return code: 0
 stdout: (none)
 stderr:
     fatal: remote error: 
       The unauthenticated git protocol on port 9418 is no longer supported.
     Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
     
 Check the log at /home/runner/.cache/pre-commit/pre-commit.log
 /usr/bin/tar --use-compress-program zstd -T0 -cf cache.tzst -P -C /home/runner/work/bfly-infra/bfly-infra --files-from manifest.txt
 Cache saved successfully
 /opt/hostedtoolcache/Python/3.10.1/x64/bin/pre-commit run --show-diff-on-failure --color=always --all-files
 [INFO] Initializing environment for git://github.com/antonbabenko/pre-commit-terraform.
 An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'fetch', 'origin', '--tags')
 return code: 128
 expected return code: 0
 stdout: (none)
 stderr:
     fatal: remote error: 
       The unauthenticated git protocol on port 9418 is no longer supported.
     Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
     
 Check the log at /home/runner/.cache/pre-commit/pre-commit.log
 Error: The process '/opt/hostedtoolcache/Python/3.10.1/x64/bin/pre-commit' failed with exit code 3
```

**Comment by asottile** [comment]
dupe #137

tl;dr from that: this is not a pre-commit problem but your configuration
